### ISSUE TITLE
Will the code continue running when the computer is put to sleep?

### ISSUE THREAD

**Issue by Mahesh2877** [aakarshg/bookmyshow#3]
Hello aakarshg@,
Thanks for writing this awesome program.

I was curious to know if it will continue running if I put the computer to sleep(not shutdown). Do you know if the code will continue running in the backend and send the email when the condition satisfies?

**Comment by Ajan** [comment]
No, the code will not continue running in the background if you put your computer to sleep. In sleep mode, most processes, including Python scripts, are paused. The script won't be able to check for ticket availability or send email notifications while the computer is asleep.

But, if that's your goal; you can deploy the script on a server or a cloud-based platform that allows for continuous execution, such as a virtual private server (VPS) or a serverless computing service.

Another approach would be to schedule the script to run periodically using a task scheduler or a cron job on a server or another device that remains powered on and connected to the internet (maybe a Raspberry Pi). This way, the script will be able to execute at the scheduled intervals and send email notifications when the condition is satisfied, regardless of the state of your personal computer.
